### DISCUSSION TITLE
exclude_docs config set up to include .github/ files still gives 404

### DISCUSSION THREAD

**Discussion by juliabelostotckaia** [mkdocs/mkdocs#3620] [Q&A]
After setting up exclude_docs in mkdocs.yml
```
exclude_docs: |
     !.github/
```
The warning messages `Doc file '*/README.md' contains a link to '*/.github/CODEOWNERS' which is excluded from the built site.` disappeared. But the deployed site still gives 404 error for all pages with `.github` in path. It is deployed to github pages.
Locally with ` mkdocs serve --clean` everything works.

**Comment by pawamoy** [comment]
The issue here is that your README links to a file that is not within your docs folder (`.github/CODEOWNERS`), and therefore cannot be included in the final site. Since it's not included in the final site, MkDocs warns you that links from the README will be broken in the final site. Just hiding these warnings won't make the code owners page appear in the site.

The only solutions I see for you here are to:

1. use an absolute link (instead of a relative one) to the CODEOWNER file on GitHub
2. or maintain a copy of the CODEOWNER file in the docs folder, and link to that file from the README.

I believe there's nothing actionable here for MkDocs so I'll move this as a Q/A discussion instead.
